CI/circle_parallel.sh

```diff
@@ -89,24 +89,6 @@ elif [ "$NODE_INDEX" = "3" ]; then
   (cd samples/client/petstore/javascript-es6 && mvn integration-test)
   (cd samples/client/petstore/javascript-promise-es6 && mvn integration-test)
 
-elif [ "$NODE_INDEX" = "4" ]; then
-  echo "Running node $NODE_INDEX ..."
-
-  #wget https://www.python.org/ftp/python/3.8.9/Python-3.8.9.tgz
-  #tar -xf Python-3.8.9.tgz
-  #cd Python-3.8.9
-  #./configure --enable-optimizations
-  #sudo make altinstall
-  pyenv install --list 
-  pyenv install 3.7.12
-  #pyenv install 2.7.14 #python2 no longer supported
-  pyenv global 3.7.12
-
-  (cd samples/openapi3/client/petstore/python && mvn integration-test)
-  (cd samples/openapi3/client/petstore/python-pydantic-v1 && mvn integration-test)
-  (cd samples/openapi3/client/petstore/python-aiohttp && mvn integration-test)
-  (cd samples/openapi3/client/petstore/python-pydantic-v1-aiohttp && mvn integration-test)
```

Comment:
i might have missed it. where did these tests go/relocate?

Reply:
It's in the commit message:

> [ci: Stop running Python integration tests in Circle CI](https://github.com/OpenAPITools/openapi-generator/pull/17509/commits/1a256e0b1368fb706904c616db411d3ad54a4563) 
>
> These Python clients are all tested with Github Actions already, which
> is faster, more cleanly separated and only runs on modifications.
>
> I'm not cleaning up the pydantic-v1 client, I expect it to be removed soon anyway.

Reply:
> These Python clients are all tested with Github Actions already, which
is faster, more cleanly separated and only runs on modifications.

from what I recall, I don't think that's the case as python petstore (fake) tests cover a lot more model tests while echo spec doesn't have those schema definitions in the echo_api.yaml

Reply:
These four `samples/openapi3/client/petstore/python*` sample folders are all tested with Github Actions workflows in `.github/workflows`.

Reply:
ah I see. completely forgot that.

can you please resolve the merge conflicts and i'll merge it right away? thanks again

Reply:
Done, thank you!